context/brand-context/springernature/scss/60-utilities/container.scss

```diff
@@ -0,0 +1,7 @@
+.u-container {
+	@include u-container();
+	background-color: $context--container-background-color;
+	border: 1px solid $context--container-border-color;
+	box-shadow: $context--container-box-shadow;
+	padding: 0 spacing(16);
```

Comment:
utilities shouldn't work like this. You should just include the mixin here, ALL the style configuration should be in the mixin. The idea is that you can get all the functionality of the utility class via the mixin. So this should be
```
.u-container {
	@include u-container();
}
```